ui/src/Components/ProfilePage.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Col, Form, FormGroup, Label } from 'reactstrap';
+
+export default class ProfilePage extends React.Component {
+  constructor(props){
+    super(props);
+
+    this.state = {
+      loading: true,
+      profile: null,
+      props: props,
+    }
+  }
+
+  async componentDidMount()
+  {
+    // const { match: { params } } = this.props;
```

Comment:
Seems like these comments were to try something out and we went with something else. We can remove these if they're not needed.

Reply:
Good catch